scripts/linux_build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -xe
+cd mnt/test
+mkdir -p build && cd build
+cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -EDGE_SDK_BUILD_EXAMPLES=ON ..
+make -j8
+cd ..
+CPP_TEST_SOURCE_AUTHENTICATION=build/olp-cpp-sdk-authentication/test
+CPP_TEST_SOURCE_CORE=build/olp-cpp-sdk-core/tests
+CPP_O="$(find build/* | grep cpp.o)"
+tar -czf artifacts.tar.gz --exclude='*.git' \
+    "build/lib" \
+    `find build -type f -executable` \
+set +e #Don't fail job on failed tests
```

Comment:
seems only DefaultCache.bad_path fails. I would say filter this out in the script and make script failing in case tests failed, additionally create a ticket for the failing tests so we can handle them asap.